src/js/helpers/test-helper.js

```diff
@@ -0,0 +1,9 @@
+import isEqual from 'lodash/isEqual'
+export class TestHelper {
```

Comment:
we already have such class, see

https://github.com/tokend/web-client/blob/master/src/test/test-helper.js

Reply:
ok